autoapi/_mapper.py

```diff
@@ -518,12 +527,18 @@ def _hide_yo_kids(self):
         for module in self.paths.values():
             if module["all"] is not None:
                 all_names = set(module["all"])
+                _trace_visibility(self.app, f"{module['full_name']}: Found __all__ =")
+                for n in all_names:
+                    _trace_visibility(self.app, f"    {n}")
                 for child in module["children"]:
                     if child["qual_name"] not in all_names:
+                        _trace_visibility(self.app, f"Hiding {child['full_name']}, as {child['qual_name']} not in __all__")
                         child["hide"] = True
             elif module["type"] == "module":
+                _trace_visibility(self.app, f"Testing if any children of {module['full_name']} have already been documented")
                 for child in module["children"]:
                     if "original_path" in child:
+                        _trace_visibility(self.app, f"Hiding {child['full_name']} as documented at {child['original_path']}")
```

Comment:
These items are hidden not because they're documented elsewhere (in fact they may not be documented anywhere) because we don't consider objects imported from the local package into a module to be part of the public API. Usually those imports are used in the implementation of a class or function in that module instead.